src/app/filter.pipe.ts

```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Todo } from './todo';
+import { ListComponent } from './list.component';
```

Comment:
why import list component and todo here ?

Reply:
forget to removed them sir!will removed!